Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.yml

```
@@ -0,0 +1,60 @@
+category: Authentication
+commonfields:
+  id: ThycoticDSV
+  version: -1
+configuration:
+- display: Server URL to DSV (e.g. https://example.com)
+  name: url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Client ID for client-credential authenticate
+  hidden: false
+  name: client_id
+  required: true
+  type: 0
+- display: Client Secret for client-credential authenticate
```

Will the user know where to get the client ID and client secret?
```suggestion
- display: Client Secret
```

---

From the documentation for the DSV product, these parameters are created using CLI commands.